src/base/os.lua

```diff
@@ -11,7 +11,6 @@
 --
 
 	premake.override(os, "execute", function(base, cmd)
-		cmd = path.normalize(cmd)

```

Comment:
i did i quick look on path.normalize implementation and i couldn't understand, what it's intention is. So if I'd fix it - I could potentially break other places.
I'll take closer look on it path.normalize.